baybe/utils/sampling_algorithms.py

```diff
@@ -31,6 +32,9 @@ def farthest_point_sampling(
               largest distance. If only a single point is requested, a deterministic
               choice is made based on the point coordinates.
             * ``"random"``: The first point is selected uniformly at random.
+        random_tie_break: Determines if points are chosen deterministically or randomly
+            in equidistant situations. If ``True``, a random point is selected from the
+            candidates. If ``False``, the first point is selected deterministically.
```

Comment:
```suggestion
            candidates, otherwise the first point is selected.
```

just to make sure, is `argmax` actually deterministically returning the `first` point?

Reply:
Yes, first occurrence in this case. But my concern here is rather that the way it is written is a bit ambiguous, because it does *not* return a random point / first point from the *candidates* but rather from the farthest ones in the current iteration. Can we somehow rephrase this to make that part clear?

Reply:
It says in equidistant situations. I will add that for non-equidistant situations, the point with the largest minimum distance is picked

Reply:
thx